### ISSUE TITLE
update for Spring Boot 2.6

### ISSUE THREAD

**Issue by SingingBush** [f-lopes/spring-mvc-test-utils#127]
for some reason it seems this isn't working with Spring Boot 2.6 and Java 17. Perhaps worth updating spring dependencies

**Comment by SingingBush** [comment]
seen now that Spring in the develop branch is recent. perhaps worth doing a new release though.

**Comment by f-lopes** [comment]
Thanks for your report. Could you please explain the error?

I have successfully tested the `spring-mvc-test-utils:3.1.0` version with Spring Boot 2.6.1 and Java 17: https://github.com/f-lopes/spring-mvc-test-utils/tree/issue/%23127-spring-boot-2.6-java-17

You can test it using this command:
```shell
docker run --rm -it -v ~/.m2/:/root/.m2/ -v $(pwd):/workspace -w /workspace maven:3.8.4-openjdk-17-slim mvn clean verify
```

Spring Boot `2.6` is using the `5.3.13` version of `spring-webmvc` whereas `spring-mvc-test-utils:3.1.0` depends on the `5.3.3`.
There isn't a lot of differences except some bugfixes to be honest.

**Comment by f-lopes** [comment]
I just saw your reply. Although the `develop` branch contains dependencies updates, the `spring-mvc-test-utils:3.1.0` released version depends on `spring-web-mvc` `5.3.3` which is not so old: https://mvnrepository.com/artifact/io.florianlopes/spring-mvc-test-utils/3.1.0

**Comment by f-lopes** [comment]
By the way, a new release containing dependencies updates is planned.
